### ISSUE TITLE
Ending Game when Bubbles Hit the Bottom

### ISSUE THREAD

**Issue by cjfalk32** [Corey56/RingLeader#8]
As the levels get harder the refresh rate gets faster. However, if the block of bubbles hit the bottom the game does not end. This does not add to the score but, does not give the user any chance to lose other than getting hit. To increase the difficulty of the game make it so the game ends once the block of bubbles hit the bottom of the screen.

**Comment by antonrowe** [comment]
I agree. The game would be much more intuitive and thrilling if the game only ended if the player was struck by a falling balloon.
